### PR TITLE
LPS-155899  title and description can be blank on locale, return title and description from default locale

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
https://issues.liferay.com/browse/LPS-155899

- LPS-155899 if title is blank and locale is different from default locale, return title from default locale
- LPS-155899 return description on default locale if the description is black on asked locale

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

File: modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/search/spi/model/result/contributor/DLFileEntryModelSummaryContributor.java [hunk 1]
```diff
@@ -39,6 +40,9 @@
 	public Summary getSummary(
 		Document document, Locale locale, String snippet) {
 
+		Locale defaultLocale = LocaleUtil.fromLanguageId(
+			document.get("defaultLanguageId"));
```
Comment (adolfopa) [inline modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/search/spi/model/result/contributor/DLFileEntryModelSummaryContributor.java:44]: Can you use the `Field.DEFAULT_LANGUAGE_ID` constant here?

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Review by adolfopa — APPROVED** [review]

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by adolfopa** [comment]
ci:reevaluate:1397401_259

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:forward

**Pull request closed without merging**